src/pages/workspace/accounting/qbo/import/QuickbooksLocationsPage.tsx

```diff
@@ -20,8 +20,12 @@ function QuickbooksLocationsPage({policy}: WithPolicyProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const policyID = policy?.id ?? '';
-    const {syncLocations, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const {syncLocations, pendingFields, reimbursableExpensesExportDestination, nonReimbursableExpensesExportDestination} = policy?.connections?.quickbooksOnline?.config ?? {};
     const isSwitchOn = Boolean(syncLocations && syncLocations !== CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE);
+    const canImportLocation =
+        reimbursableExpensesExportDestination === CONST.QUICKBOOKS_REIMBURSABLE_ACCOUNT_TYPE.JOURNAL_ENTRY &&
+        nonReimbursableExpensesExportDestination !== CONST.QUICKBOOKS_NON_REIMBURSABLE_EXPORT_ACCOUNT_TYPE.VENDOR_BILL;
```

Comment:
If accounts can be missing and the non-reimbursable/reimbursable accounts can be empty, should update the condition to allow importing locations in that case? cc @trjExpensify

Reply:
It was a pusher issue, I don't think we need to do anything.

Reply:
If there aren't any accounts imported from QBO we have this empty state: 

<img width="2039" alt="image" src="https://github.com/Expensify/App/assets/16232057/331ae851-7869-46b1-a1fa-a0e9d3c5cbe4">

I don't really see the value in allowing locations import if an export type hasn't been selected, though it should be an edge case. 

There's also an issue to fix pusher being flaky sometimes with sending the `jobDone` sync update, so hopefully we stamp that out. I believe Yuwen is working on it.